values.yaml

```diff
@@ -67,7 +69,7 @@ secret:
 
 service:
   enabled: true
-  type: ClusterIP
+  type: NodePort
```

Comment:
I'm not sure that it's the correct default service type. In my case, I use this service behind ingress and NodePorts is disabled in my cluster. Can you revert default here please